Fixed default swift version

gtrop1 opened the pull request.

cnoon commented on CHANGELOG.md:
```diff
@@ -40,6 +40,8 @@ All notable changes to this project will be documented in this file.
 #### Removed
 
 #### Fixed
+- Default Swift version now correctly set to 4.2.
```
How about: `- Issue where Swift version was incorrectly set to 5.0 when it should have been 4.2.`?

cnoon reviewed (changes requested):
Let's just wrap all the Info.plist and podspec changes into this as well.

cnoon commented on CHANGELOG.md:
```diff
@@ -43,6 +43,14 @@ All notable changes to this project will be documented in this file.
+## [5.2.1](https://github.com/Nike-Inc/Willow/releases/tag/5.2.1)
+
+#### Fixed
+- Issue where Swift version was incorrectly set to 5.0 when it should have been 4.2..
```
Extra `.` at the end.

cnoon reviewed (changes requested):
One small comment back.

gtrop1 committed:
Default Swift version now correctly set to 4.2

Addressed issue where Swift version was incorrectly set to 5.0 when it should have been 4.2.

cnoon reviewed (approved):
Looks good!
